### PR TITLE
Lots of PHPDoc fixes

### PR DESCRIPTION
**Pull request by rrran**
Just some PHPDoc fixes

### CODE REVIEW

File: phpseclib/Crypt/Common/Keys/OpenSSH.php [hunk 1]
```diff
@@ -103,6 +103,7 @@ public static function load($key, $type)
      * Returns the comment for the key
      *
      * @access public
+     * @param $key
```
Comment (GrahamCampbell) [inline phpseclib/Crypt/Common/Keys/OpenSSH.php]: This is invalid syntax.

Comment (GrahamCampbell) [reply phpseclib/Crypt/Common/Keys/OpenSSH.php]: Missing type data.

File: phpseclib/Crypt/RC4.php [hunk 2]
```diff
@@ -256,7 +255,7 @@ public function decrypt($ciphertext)
 
     /**
      * Encrypts a block
-     *
+     * @return null
```
Comment (GrahamCampbell) [inline phpseclib/Crypt/RC4.php]: 👎

File: phpseclib/Crypt/DSA.php [hunk 3]
```diff
@@ -460,6 +460,7 @@ function sign($message, $format = 'Raw')
      * @see self::verify()
      * @access public
      * @param string $message
+     * @param $signature
```
Comment (GrahamCampbell) [inline phpseclib/Crypt/DSA.php]: Invalid syntax

File: phpseclib/Crypt/DSA.php [hunk 4]
```diff
@@ -213,10 +213,10 @@ static function createKey()
      * Loads a public or private key
      *
      * Returns true on success and false on failure (ie. an incorrect password was provided or the key was malformed)
-     *
+     * @return boolean
```
Comment (GrahamCampbell) [inline phpseclib/Crypt/DSA.php]: bool

File: phpseclib/File/ASN1.php [hunk 5]
```diff
@@ -831,8 +831,8 @@ public static function encodeDER($source, $mapping, $special = [])
      * @param string $source
      * @param array $mapping
      * @param int $idx
+     * @param array $special
      * @return string
-     * @throws \RuntimeException if the input has an error in it
```
Comment (terrafrost) [inline phpseclib/File/ASN1.php]: Why was this removed?

File: phpseclib/Crypt/Common/AsymmetricKey.php [hunk 6]
```diff
@@ -180,6 +180,7 @@ public function __construct()
     /**
      * Tests engine validity
      *
+     * @returns boolean
```
Comment (terrafrost) [inline phpseclib/Crypt/Common/AsymmetricKey.php]: Should be `@return` (singular)

**Comment by terrafrost** [comment]
I went ahead and fixed the issues that GrahamCampbell found. There were multiple instances of each issue and those multiple instances should be fixed as well.

The one thing I didn't change was the removal of the `@throws` thing. I'm not an expert on PHPDoc and I'll give you the benefit of the doubt on that one!

**Comment by terrafrost** [comment]
The commit where I fixed the issues:

https://github.com/phpseclib/phpseclib/commit/ec579bd9ed016fad7c417e5c35b303949e3d9555
